### ISSUE TITLE
Handle maintenance release targets differently or disallow dots in package names

### ISSUE THREAD

**Issue by bgeuken** [openSUSE/open-build-service#1449]
Current OBS code is adding a suffix to packages updates that get submitted as maintenance request. The sufix looks like ".#{release_target_name}".
OBS then uses the first dot for extracting the release target from such packages. See https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/project.rb#L1599 .

Since dots are allowed in OBS this possibly can cause confusion and bugs in OBS.

**Comment by mlschroe** [comment]
I don't think we can/should disallow dots, there are lots of packages that have a dot in them.

**Comment by bgeuken** [comment]
Me neither. I think we shouldn't bind such additional information to package names. Be it release targets or things like "_patchinfo", "_product", "_link", ...

**Comment by bgeuken** [comment]
One of the side effects: #1758
